tf2onnx/convert.py

```diff
@@ -145,10 +145,10 @@ def main():
                              output_names=outputs,
                              inputs_as_nchw=args.inputs_as_nchw)
 
-    model_proto = g.make_model("converted from {}".format(model_path))
-
-    logger.info("")
-    model_proto = GraphUtil.optimize_model_proto(model_proto)
+    onnx_graph = optimizer.optimize_graph(g)
+    if not onnx_graph:
```

Comment:
optimizer.optimize_graph should always return a valid graph, no need to check None here.

you may want to add a None check at https://github.com/onnx/tensorflow-onnx/blob/d239d1cc5f73d45be93b2f2160d408253538871d/tf2onnx/optimizer/__init__.py#L50
something like ```graph = opt.optimize(current) or graph```

and we can skip None check here.

Reply:
done.